auth/README.md

```diff
@@ -315,24 +315,37 @@ redefine a string to change it, for example:
 
 ### OAuth Scope Customization
 
+#### Google
+By default, FirebaseUI requests the `email` and `profile` scopes when using Google Sign In. If you
+would like to request additional scopes from the user, add a string array resource to your
+application like this:
```

Comment:
Where do I add this? A new element in an existing file? Or is it a new file? If so, does the name of the file matter? Or is it only about the array's name?